go.mod

```diff
@@ -17,8 +17,15 @@ require (
 	github.com/google/go-cmp v0.3.0
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
+<<<<<<< HEAD
 	github.com/openshift/api v3.9.1-0.20191015204012-3f47c9a8fa7f+incompatible
 	github.com/openshift/library-go v0.0.0-20190927184318-c355e2019bb3
+=======
+	github.com/libvirt/libvirt-go v5.8.0+incompatible
+	github.com/openshift/api v3.9.1-0.20190905013149-e6c3eeae444f+incompatible
+	github.com/openshift/library-go v0.0.0-20190613200606-e617f832835d
+	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709 // indirect
+>>>>>>> Enhance cluster-ingress-operator to support libvirt
```

Comment:
merge conflict, committed here against instead or resolving it.

Reply:
I'm sorry for my carelessness, I will fix it. Thanks.